Classes/Controller/BackendServiceController.php

```diff
@@ -618,4 +619,34 @@ public function generateUriPathSegmentAction(string $contextNode, string $text):
         $slug = $this->nodeUriPathSegmentGenerator->generateUriPathSegment($contextNode, $text);
         $this->view->assign('value', $slug);
     }
+
+    /**
+     * Rebase user workspace to current workspace
+     *
+     * @param string $targetWorkspaceName
+     * @return void
+     */
+    public function rebaseWorkspaceAction(string $targetWorkspaceName): void
+    {
+        $contentRepositoryId = SiteDetectionResult::fromRequest($this->request->getHttpRequest())->contentRepositoryId;
+        $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
+
+        $command = RebaseWorkspace::create(WorkspaceName::fromString($targetWorkspaceName));
+        try {
+            $contentRepository->handle($command)->block();
+        } catch (\Exception $exception) {
+            $error = new Error();
+            $error->setMessage($error->getMessage());
+
+            $this->feedbackCollection->add($error);
+            $this->view->assign('value', $this->feedbackCollection);
+            return;
+        }
+
+        $success = new Success();
+        $success->setMessage(sprintf('Successfully synced User workspace'));
```

Comment:
Is that something that needs to be localized?

Reply:
Indeed will take care later!

Reply:
It might be that those messages are not a flashmessage but just show up in the console as debug info in which case translation is irrelevant

Reply:
We should probably then also localize all of the other $success->setMessage calls in the Controller to be consistent